api/tests/src/api/opportunities_v1/test_opportunity_route_search.py

```diff
@@ -710,6 +710,47 @@ def test_search_filters_200(
     ):
         call_search_and_validate(client, api_auth_token, search_request, expected_results)
 
+    @pytest.mark.parametrize(
+        "search_request, expected_status_code",
+        [
+            # Post Date
+            (get_search_request(post_date={"start_date": None}), 200),
+            (get_search_request(post_date={"end_date": None}), 200),
+            (get_search_request(post_date={"start_date": "2020-01-01"}), 200),
+            (get_search_request(post_date={"end_date": "2020-02-01"}), 200),
+            (get_search_request(post_date={"start_date": None, "end_date": None}), 200),
+            (get_search_request(post_date={"start_date": "2020-01-01", "end_date": None}), 200),
+            (get_search_request(post_date={"start_date": None, "end_date": "2020-02-01"}), 200),
+            (get_search_request(post_date={"start_date": "2020-01-01", "end_date": "2020-02-01"}), 200),
+            (get_search_request(post_date={"start_date": "I am not a date"}), 422),
+            (get_search_request(post_date={"end_date": "I am not a date"}), 422),
```

Comment:
A minor nitpick - could you add a small bit of variance in the error cases? Just something like `"123-456-789"` or `5` for the values on a few of these? We don't need to extensively test it as its ultimately another library, but just helps if any regressions happen in those libraries I find.

Reply:
This should be in place now with the latest commit. Let me know what you think. Thank you!